frontend-react/src/hooks/UsePagination.test.ts

```diff
@@ -590,4 +596,40 @@ describe("usePagination", () => {
         expect(mockFetchResults2).toHaveBeenLastCalledWith("1", 61);
         expect(result.current.paginationProps?.currentPageNum).toBe(1);
     });
+
+    test("Calls appInsights.trackEvent with page size and page number.", async () => {
```

Comment:
This is out of the scope of your pull request, but I think the tests in this file could be cleaned up a bit.  BUT as we've already discussed, we can refactor the existing usePagination hook to just leverage react-query to simplify a lot of this stuff -- so cleanup here can definitely wait!